.gitlab-ci.yml

```diff
@@ -138,37 +127,47 @@ build: ## build data.table sources as tar.gz archive
 #  tags:
 #    - macosx
 
-test-rel-lin: ## most comprehensive tests, force all suggests, also integration tests, using gcc -O3 -flto -fno-common -Wunused-result
+## most comprehensive tests
+# force all suggests
+# flags: gcc -O3 -flto -fno-common -Wunused-result
+# tests for compilation warnings
+# measure memory usage during tests
+test-rel-lin:
```

Comment:
`lin` is `linux` here? I would just write out `linux` I don't see much benefit to saving 2 characters

Reply:
Yes, it is 3 letters so it aligns better when printing to win and mac. Maybe lnx then?

Reply:
That would be less ambiguous, yes